common/src/main/java/com/microsoft/identity/common/exception/ErrorStrings.java

```diff
@@ -350,4 +350,34 @@ private ErrorStrings() {
      */
     public static final String KEY_NOT_FOUND = "key_not_found";
 
+    /**
+     * AccountMode in configuration is set to multiple. However, the device is marked as shared (which requires single account mode).
+     */
+    public static final String CANNOT_INITIALIZE_MULTIPLE_ACCOUNT_PCA_ON_SHARED_DEVICE_ERRORCODE = "cannot_initialize_multiple_account_pca_on_shared_device";
```

Comment:
Error constant name seems too long, consider shortening the constant name like `MULTIPLE_ACCOUNT_INIT_FAIL_ERROR_CODE` for all of them below